### PR TITLE
fix: user created timestamp not cast to Carbon

### PR DESCRIPTION
**Pull request by luchaos**
Do not set timestamps to false on user object - this prevents datetime casting for Created and Updated attributes.

Change ticket `isAllowedToSubmitTickets()` check.

### CODE REVIEW

File: app/Helpers/database/ticket.php [hunk 1]
```diff
@@ -15,35 +15,36 @@
...
     }
 
-    $cacheKey = CacheKey::buildUserCanTicketCacheKey($user);
+    $cacheKey = CacheKey::buildUserCanTicketCacheKey($username);
 
-    $value = Cache::get($cacheKey);
-    if ($value !== null) {
-        return $value;
+    if (Cache::has($cacheKey)) {
+        return Cache::get($cacheKey);
```
Comment (Jamiras) [inline app/Helpers/database/ticket.php:27]: Doesn't this cause a double lookup?

Comment (Jamiras) [reply app/Helpers/database/ticket.php:27]: Without the join against Activity, we could probably just eliminate caching of this altogether. What about something like this?
```
function isAllowedToSubmitTickets(string $username): bool
{
    $user = User::firstWhere('User', $username);
    if (!$user || $user->Created->diffInDays() < 1) { 
        return false;
    }

    return PlayerGame::where('user_id', $user->id)
        ->where('time_taken', '>', 5)
        ->exists();
}
```

Comment (luchaos) [reply app/Helpers/database/ticket.php]: very nice - changed in https://github.com/RetroAchievements/RAWeb/pull/1954/commits/8bbe110fd76f6fabf42f8596df0c39d3a09e1186

Comment (Jamiras) [reply app/Helpers/database/ticket.php]: Should also remove the code for `buildUserCanTicketCacheKey`

Comment (luchaos) [reply app/Helpers/database/ticket.php]: removed in https://github.com/RetroAchievements/RAWeb/pull/1954/commits/ea238e5d0169684a295cd71f79b748907b76a9c9

**Review by Jamiras — APPROVED** [review]

**Pull request merged**